ts/pages/account/dashboard.tsx

```diff
@@ -83,9 +84,11 @@ interface PendingAction {
     amount: number;
 }
 
-// TODO(johnrjj) - After going to a pool that has been staked in, hitting the 'Remove' button,
-// and running the transaction, the balances do not update, even after a good amount of time (30min+ etc)
-// Is there a way we can query web3 directly?? (tl;dr 'Pending' removes are flaky/not sure if they are working)
```

Comment:
@johnrjj can you confirm that this is not an issue anymore? It works fine for me but I'm not sure I fully understand what the problem was.

Reply:
This must've been outdated -- everything looks good now! :)